docs/_config.yml

```diff
@@ -0,0 +1,56 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+title: Sfall
```

Comment:
Wasn't it was always `sfall` all-lowercase? :) @NovaRain

Reply:
I don't know.

Reply:
AFAIK Timeslip seems always using all-lowercase `sfall` instead of capitalized "Sfall" or "sFall" when it comes to the name.

Reply:
lowercased